modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -2548,6 +2548,13 @@ void GDScriptAnalyzer::reduce_call(GDScriptParser::CallNode *p_call, bool p_is_a
 			mark_lambda_use_self();
 		}
 
+#ifdef DEBUG_ENABLED
+
```

Comment:
I'd remove this empty line, doesn't really help readability here IMO.

Reply:
Done